ci/parse-changelog

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+cd "$(dirname $0)"
+
+docker run --rm \
```

Comment:
Thanks, I did it this way in `ldap-sync` after realizing what I did in `conjur` was silly but hadn't had time to go back and fix it yet.

Reply:
No worries, thanks for taking a look.